config/toml.go

```diff
@@ -55,4 +55,8 @@ keep-recent = {{ .StateStore.KeepRecent }}
 # default to 60 seconds
 prune-interval-seconds = {{ .StateStore.PruneIntervalSeconds }}
 
+# ImportNumWorkers defines the number of goroutines used during import
+# defaults to 1
+import-num-workers = {{ .StateStore.ImportNumWorkers }}
```

Comment:
One thing we've considered is commenting out these configs by default, and setting the default in the codebase.  This allows us to change the default.